src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/ReportedAdjudicationServiceTest.kt

```diff
@@ -336,19 +452,6 @@ class ReportedAdjudicationServiceTest {
       whenever(draftAdjudicationRepository.save(any())).thenReturn(savedDraftAdjudication)
     }
 
-    @ParameterizedTest
-    @CsvSource(
-      "ACCEPTED, 99",
-      "REJECTED, 999"
-    )
-    fun`create draft from reported adjudication responds with 400 for status`(status: ReportedAdjudicationStatus, id: Long) {
```

Comment:
I might be blind, but have you replaced this test?

Reply:
Test being written

Reply:
TODO